### PR TITLE
Tool 574 update pkcs12

### PR DESCRIPTION
**Pull request by lpusok**

### CODE REVIEW

File: pkcs12/pkcs12.go [hunk 1]
```diff
@@ -306,6 +346,66 @@ func DecodeAllCerts(pfxData []byte, password string) (certificates []*x509.Certi
 	return
 }
 
+// DecodeCodesignCertificates returns an array of certificate and private keys
+// Used to parse p12 file containing multiple codesign identities (exported from macOS Keychain)
+func DecodeCodesignCertificates(pfxData []byte, password string) (identities []SigningIdentity, err error) {
```
Comment (lpusok) [inline pkcs12/pkcs12.go]: DecodeAll or DecodeAllCertsAndPrivateKeys

Comment (lpusok) [reply pkcs12/pkcs12.go]: remove DecodeAllCerts

Comment (godrei) [reply pkcs12/pkcs12.go]: is not it equivalent to `DecodeChain `: https://github.com/bitrise-io/go-utils/pull/94/files#diff-daede74dc1a2aed976de72338be4792dR250

Comment (lpusok) [reply pkcs12/pkcs12.go]: no

File: pkcs12/pkcs12.go [hunk 2]
```diff
@@ -208,65 +231,82 @@ func convertAttribute(attribute *pkcs12Attribute) (key, value string, err error)
...
 	if privateKey == nil {
-		return nil, nil, errors.New("pkcs12: private key missing")
+		return nil, nil, nil, errors.New("pkcs12: private key missing")
 	}
 
 	return
 }
 
+// SigningIdentity is a certificate with a matching private key
+type SigningIdentity struct {
```
Comment (godrei) [inline pkcs12/pkcs12.go]: we should figure out a better name for this
what about `CertificatePrivateKeyBundle` ?

Comment (godrei) [reply pkcs12/pkcs12.go]: or simply return a list of certificates and private keys

**Pull request merged**